demonstrations/tutorial_barren_gadgets.py

```diff
@@ -0,0 +1,332 @@
+r"""
+Perturbative Gadgets for Variational Quantum Algorithms  
+==========================================
+
+*Author: Simon Cichy*
+
+In this tutorial, we will explore the application of perturbative gadgets in 
+variational quantum algorithms to outgo the issue of cost function dependent
+barren plateaus, as proposed in Ref.[#cichy2022]

```

Comment:
```suggestion
barren plateaus, as proposed in Ref.[#cichy2022]_
```
This will ensure it automatically links to the Bibliography at the bottom